### ISSUE TITLE
failed to update cadfaelbrew from GitHub

### ISSUE THREAD

**Issue by victortretyak** [SuperNEMO-DBD/cadfael-installer#1]
Cadfael installation is failed:
$ ./cadfael-installer
[cadfael:info] Detected 'Linux' Operating System
[cadfael:info] Distribution 'Ubuntu (14.04)'
[cadfael:info] in doCheckSystemSoftware Ubuntu-14.04
[cadfael:info] System supported
[cadfael:info] Checking system software for Ubuntu 14.04
[cadfael:info] Checking for install of deb 'build-essential': 11.6ubuntu6 install ok installed
[cadfael:info] Checking for install of deb 'curl': 7.35.0-1ubuntu2.5 install ok installed
[cadfael:info] Checking for install of deb 'git': 1:1.9.1-1ubuntu0.2 install ok installed
[cadfael:info] Checking for install of deb 'm4': 1.4.17-2ubuntu1 install ok installed
[cadfael:info] Checking for install of deb 'libbz2-dev': 1.0.6-5 install ok installed
[cadfael:info] Checking for install of deb 'libcurl4-openssl-dev': 7.35.0-1ubuntu2.5 install ok installed
[cadfael:info] Checking for install of deb 'libexpat-dev':  unknown ok not-installed
[cadfael:info] Checking for install of deb 'libncurses-dev':  unknown ok not-installed
[cadfael:info] Checking for install of deb 'ruby2.0': 2.0.0.484-1ubuntu2.2 install ok installed
[cadfael:info] Checking for install of deb 'texinfo': 5.2.0.dfsg.1-2 install ok installed
[cadfael:info] Checking for install of deb 'zlib1g-dev': 1:1.2.8.dfsg-1ubuntu1 install ok installed
[cadfael:info] Checking for install of deb 'libx11-dev': 2:1.6.2-1ubuntu2 install ok installed
[cadfael:info] Checking for install of deb 'libxpm-dev': 1:3.5.10-1 install ok installed
[cadfael:info] Checking for install of deb 'libxft-dev': 2.3.1-2 install ok installed
[cadfael:info] Checking for install of deb 'libxext-dev': 2:1.3.2-1ubuntu0.0.14.04.1 install ok installed
[cadfael:info] Checking for install of deb 'libpng12-dev': 1.2.50-1ubuntu2.14.04.2 install ok installed
[cadfael:info] Checking for install of deb 'libjpeg-dev': 8c-2ubuntu8 install ok installed
[cadfael:info] Checking basic network connectivity
[cadfael:info] Cloning 'cadfaelbrew.git' into /home/victor/supernemo/cadfael-installer.git/Cadfael.git
Cloning into '/home/victor/supernemo/cadfael-installer.git/Cadfael.git'...
remote: Counting objects: 315805, done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 315805 (delta 0), reused 0 (delta 0), pack-reused 315801
Receiving objects: 100% (315805/315805), 61.29 MiB | 244.00 KiB/s, done.
Resolving deltas: 100% (237865/237865), done.
Checking connectivity... done.
[cadfael:info] setting up brew
[cadfael:info] System ruby (/usr/bin/ruby) version : 2.0
[cadfael:info] Updating cadfaelbrew
/home/victor/supernemo/cadfael-installer.git/Cadfael.git/bin/brew: 41: exec: -P: not found
ruby is /usr/bin/ruby: not found
[cadfael:error] failed to update cadfaelbrew from GitHub

What is wrong?

**Comment by drbenmorgan** [comment]
Thanks for the report - this was somehow missed in testing. The cause is that Ubuntu's underling `sh` implementation is very limited and does not provided the `-P` argument to `type`.

This is fixed upstream in SuperNEMO-DBD/cadfaelbrew@a8c5927c  

You can rerun the installer directly, though you'll need to remove the `Cadfael.git` subdirectory first. As all the bootstrapping has completed, you could also do:

```
$ cd /home/victor/supernemo/cadfael-installer.git/Cadfael.git
$ ./bin/brew update
$ ./bin/brew doctor
... This may issue warnings about PATH, but these can be ignored
$ ./bin/brew install cadfael
... If that completes ok, then
$ ./bin/brew install bayeux falaise
```

**Comment by victortretyak** [comment]
Thank you
Everything passed smoothly, without any warning.
